packages/flutter/test/services/raw_keyboard_test.dart

```diff
@@ -469,7 +469,7 @@ void main() {
           'toolkit': 'glfw',
           'keyCode': 0x04,
           'scanCode': 0x01,
-          'codePoint': 0x10,
+          'charactersIgnoringModifiers': 'a',
```

Comment:
how are 0x10 and 'a' related?

Reply:
I assume they're not -- this is test data. My assumption stems from the fact that a key-event for a tap on the 'a' key is something likely, whereas I'm not aware of any keyboard that'll emit keydown events for `0x10` (DLE: "data link escape" according to my local ASCII table).

That said, I'm sure that at some point such a keyboard has existed and I'd be very curious to know what the value would be according to Apple's APIs. `0x1b` (ESC), on the other hand, seems like something we could easily test their APIs against to answer what their APIs currently return.

Reply:
They are not. This test only really cares about modifiers. The rest of the key data is there just to show all the fields in object. IIRC, 0x10 is just a random value.

Reply:
In that case, this entire change appears to have had no practical impact on the tests at all, which is quite concerning! We should make sure to test this code.

Reply:
This particular test is not affected by the codePoint since it's only testing for the modifiers. The tests for key translations are below. 

I think the keyCode and codePoint from the previous patch were just random numbers but I changed them now to mimic an actual 'A' key press in all fields.